docs/reference/commandline/daemon.md

```diff
@@ -446,6 +446,10 @@ these defaults are not set, `ulimit` settings will be inherited, if not set on
 `docker run`, from the Docker daemon. Any `--ulimit` options passed to 
 `docker run` will overwrite these defaults.
 
+Please notice `nproc` in `ulimit` is special as it is designed by Linux to
```

Comment:
@resouer Thanks for the contribution. Some of the constructions are awkward. Also, a bit more context will help readability.

--> https://gist.github.com/moxiegirl/ab05af352008555bfb3d

Be careful setting `nproc` with the `ulimit` flag as `nproc` is designed by Linux to
set the maximum number of processes available to a user, not to a container. For details
please check the [run](run.md) reference.